src/styles/blocks/_cards.scss

```diff
@@ -0,0 +1,135 @@
+.container {
+  max-width: 1100px;
+  margin: 0 auto;
+  padding: 84px 8px;
+
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: var(--main-color);
+
+  box-sizing: border-box;
```

Comment:
uncomment your `*` selector rule, to make it the same for all elements

Reply:
Thanks, I remove all box-sizing